.github/workflows/integration-test.yaml

```diff
@@ -19,9 +19,18 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: '21.x'
+      - name: "Update Genesis block number"
+        run: | 
+          source ./.env.IntegrationTest
+          echo "$UPDATE_GENESIS_BLOCK_QUERY" > temp.sql
+          PGPASSWORD=$DB_ADMIN_USER_SECRET psql -h localhost -p 5432 -d $DB_NAME -U $DB_ADMIN_USER_NAME -f temp.sql
       - name: "Install newman"
         run: npm install -g newman
+      - name: "Install rosetta-cli"
+        run: curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s
       - name: "Run tests"
         run: newman run ./postmanTests/rosetta-java.postman_collection.json -e postmanTests/Rosetta-java-env.postman_environment.json -r cli
+      - name: "Run rosetta check:data tests"
+        run: ./bin/rosetta-cli check:data --configuration-file ./rosetta-cli-tests/data/byron_sample.json
```

Comment:
Probably, it's worth adding to the Readme file, that we run Rosetta-cli only against the first 200 blocks within the devkit. Like this one:
https://github.com/cardano-foundation/cardano-rosetta/tree/542053f87109d883f31339194a867c57f065213c/test/check